test/IDE/complete_pattern.swift

```diff
@@ -142,3 +147,32 @@ func test<T>(x: T) {
     #^AFTER_PATTERN_IS^#
   }
 }
+
+func test_multiple_patterns1(x: Int) {
+    switch (x, x) {
+    case (0, let a), #^MULTI_PATTERN_1^#
+    }
+}
+
+// MULTI_PATTERN_1: Begin completions
+// FIXME: ideally we wouldn't offer 'a' here, without 'let' first: Decl[LocalVar]/Local: a[#Int#]{{; name=.+$}}
+
+func test_multiple_patterns2(x: Int) {
+    switch (x, x) {
+    case let (0, let a), (let a, 0):
+        #^MULTI_PATTERN_2^#
+    }
+}
+
+// MULTI_PATTERN_2: Begin completions
+
+func test_multiple_patterns3(x: Int) {
+    switch (x, x) {
+    case let (0, let a), (let b, 0):
+        #^MULTI_PATTERN_3^#
+    }
+}
+
+// MULTI_PATTERN_3: Begin completions
```

Comment:
…and even though this one's invalid, I'd still expect to at least see `x`. (Again, I don't mind if it's broken, but the check lines shouldn't be vacuous.)